local-volume/helm/provisioner/templates/provisioner.yaml

```diff
@@ -32,6 +32,9 @@ data:
        {{- range $val := $classConfig.blockCleanerCommand }}
          - "{{ $val -}}"{{- end}}
        {{- end }}
+       {{- if $classConfig.volumeMode }}
+       volumeMode: {{ $classConfig.volumeMode }}
```

Comment:
we have some test case yamls too. Can you add this there? (actually I think the helm tests will fail because the test yamls have not been updated with the new field)

Reply:
Aha, I guess the helm tests won't fail since we don't have the field in either source or output files.
But you are right, we need to add the field there for validation. Maybe add it in one of the cases should be enough?

Reply:
done, and I left some of them as is, to validate the volumeMode unset scenario